package.json

```diff
@@ -31,18 +31,26 @@
     "@types/relay-test-utils": "^6.0.2",
     "@typescript-eslint/eslint-plugin": "^4.1.1",
     "@typescript-eslint/parser": "^4.1.1",
+    "@zeit/next-less": "^1.0.1",
+    "@zeit/next-sass": "^1.0.1",
+    "antd": "^4.6.4",
     "babel-plugin-relay": "^10.0.1",
     "enzyme": "^3.11.0",
-    "enzyme-adapter-react-16": "^1.15.2",
+    "enzyme-adapter-react-16": "^1.15.5",
```

Comment:
There is no need for libraries enzyme and enzyme-adapter-react. We have to remove them.

Reply:
There is, enzyme mocks the window environment on the global object for things like navigator, document and window, i moved this adapter to /__mocks__/client.ts